server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/EndpointLogger.java

```diff
@@ -184,5 +185,9 @@
    @LogMessage(level = INFO)
    @Message(id = 10032, value = "No Security realm declared for endpoint '%s'. Using default encryption realm.")
    void noSSLContextForSni(String endpoint);
+
+   @LogMessage(level = INFO)
+   @Message(id = 10033, value = "Starting server %s without transport (use multi tenant router instead)")
```

Comment:
That's exactly my point, we only log an INFO message, but it's actually a configuration error if a server doesn't have a transport and it's not referenced in a `multi-tenancy` element either. We can't tell if the whole configuration is correct using only the server config, but I was hoping there's some way to iterate over all the servers and all the routers and log an error if one of the servers has been left out.